html/pfappserver/lib/pfappserver/Form/Config/Source/Eduroam.pm

```diff
@@ -0,0 +1,98 @@
+package pfappserver::Form::Config::Source::Eduroam;
+
+=head1 NAME
+
+pfappserver::Form::Config::Source::Eduroam
+
+=cut
+
+=head1 DESCRIPTION
+
+Form definition to create or update an Eduroam authentication source.
+
+=cut
+
+use strict;
+use warnings;
+
+use HTML::FormHandler::Moose;
+
+extends 'pfappserver::Form::Config::Source';
+with 'pfappserver::Base::Form::Role::Help';
+
+has_field 'server1_address' => (
+    type        => 'Text',
+    label       => 'Server 1 address',
+    required    => 1,
+    default     => "",
```

Comment:
No this is needed to help build the default args for building sources.

I will however comment that it is needed to avoid someone from from deleting it.